ibis/backends/impala/tests/snapshots/test_exprs/test_named_from_filter_group_by/foo.sql

```diff
@@ -1,4 +1,17 @@
-SELECT t0.`key`, sum(((t0.`value` + 1) + 2) + 3) AS `foo`
-FROM `t0` t0
-WHERE t0.`value` = 42
-GROUP BY 1
\ No newline at end of file
+SELECT
+  `t1`.`key`,
+  SUM((
```

Comment:
Can we change the newline strategy in sqlglot?

Reply:
Adds quite some vertical space in the snapshots.

Reply:
Not that i'm aware of.